codejail/tests/test_safe_exec.py

```diff
@@ -67,7 +71,7 @@ def test_printing_stuff_when_you_shouldnt(self):
     def test_importing_lots_of_crap(self):
         globs = {}
         self.safe_exec(textwrap.dedent("""\
-            from numpy import *
+            from os import *
```

Comment:
Ah.  That makes sense.  I had changed it because it introduced even more hidden assumptions about the test environment.  Given @feanil's desire to have codejail available on pypi, I want to be sure we aren't overly dependent on the specifics of edx-platform's configuration to run codejail, but I think having numpy available to the test codejail's sandbox is not an unreasonable requirement.